src/include/storage/compressed_tile.h

```diff
@@ -0,0 +1,156 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// compressed_tile.h
+//
+// Identification: src/include/storage/compressed_tile.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <mutex>
+
+#include "catalog/manager.h"
+#include "catalog/schema.h"
+#include "common/item_pointer.h"
+#include "common/printable.h"
+#include "type/abstract_pool.h"
+#include "type/serializeio.h"
+#include "type/serializer.h"
+#include "storage/tile.h"
+
+namespace peloton {
+
+namespace gc {
+class GCManager;
+}
+
+namespace storage {
+
+//===--------------------------------------------------------------------===//
+// Tile
+//===--------------------------------------------------------------------===//
+
+class Tuple;
+class TileGroup;
+class TileGroupHeader;
+class TupleIterator;
+class Tile;
+
+/**
+ * Represents a Tile.
+ *
+ * Tiles are only instantiated via TileFactory.
+ *
+ * NOTE: MVCC is implemented on the shared TileGroupHeader.
+ */
```

Comment:
Yes, Will fix the comments of the files and possibly add more comments detailing function use, parameters and uses.